youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,588 @@
+msgid "Item is not completed"
+msgstr "הפריט לא הושלם"
+
+#: youtube_dl_gui/mainframe.py:668
+msgid "Update in progress. Please wait for the update to complete"
+msgstr "עדכון בתהליך. המתן עד להשלמת העדכון"
+
+#: youtube_dl_gui/mainframe.py:716
+msgid "Logging is disabled"
+msgstr "רישום מושבת"
+
+#: youtube_dl_gui/mainframe.py:891
+msgid "Shutdown"
+msgstr "כיבוי"
+
+#: youtube_dl_gui/mainframe.py:891
+msgid "Shutting down in {0} second(s)"
+msgstr "כיבוי בעוד {0} שניות"
+
+#: youtube_dl_gui/mainframe.py:980
+msgid "No items to download"
+msgstr "אין פריטים להורדה"
+
+#: youtube_dl_gui/mainframe.py:1060
+msgid "Are you sure you want to exit?"
+msgstr "אתה בטוח שאתה רוצה לצאת?"
+
+#: youtube_dl_gui/mainframe.py:1060
+msgid "Exit"
+msgstr "יציאה"
+
+#: youtube_dl_gui/mainframe.py:1301 youtube_dl_gui/mainframe.py:1451
+msgid "Cancel"
+msgstr "בטל"
+
+#: youtube_dl_gui/mainframe.py:1450
+msgid "OK"
+msgstr "אשר"
+
+#: youtube_dl_gui/optionsframe.py:65
+msgid "Reset"
+msgstr "אפס"
+
+#: youtube_dl_gui/optionsframe.py:66
+msgid "Close"
+msgstr "סגור"
+
+#: youtube_dl_gui/optionsframe.py:72
+msgid "General"
+msgstr "כללי"
+
+#: youtube_dl_gui/optionsframe.py:73
+msgid "Formats"
+msgstr "פורמט"
+
+#: youtube_dl_gui/optionsframe.py:74
+msgid "Downloads"
+msgstr "הורדות"
+
+#: youtube_dl_gui/optionsframe.py:75
+msgid "Advanced"
+msgstr "מתקדם"
+
+#: youtube_dl_gui/optionsframe.py:76
+msgid "Extra"
+msgstr "תוספות"
+
+#: youtube_dl_gui/optionsframe.py:315
+msgid "Language"
+msgstr "שפה"
+
+#: youtube_dl_gui/optionsframe.py:318
+msgid "Filename format"
+msgstr "פורמט שם הקובץ"
+
+#: youtube_dl_gui/optionsframe.py:323
+msgid "Filename options"
+msgstr "אפשרויות שם הקובץ"
+
+#: youtube_dl_gui/optionsframe.py:324
+msgid "Restrict filenames to ASCII"
+msgstr "הגבל שמות קבצים ל- ASCII"
+
+#: youtube_dl_gui/optionsframe.py:326
+msgid "More options"
+msgstr "אפשרויות נוספות"
+
+#: youtube_dl_gui/optionsframe.py:327
+msgid "Confirm on exit"
+msgstr "אשר ביציאה"
+
+#: youtube_dl_gui/optionsframe.py:328
+msgid "Confirm item deletion"
+msgstr "אשר מחיקת פריטים"
+
+#: youtube_dl_gui/optionsframe.py:329
+msgid "Inform me on download completion"
+msgstr "יידע אותי על השלמת ההורדה"
+
+#: youtube_dl_gui/optionsframe.py:331
+msgid "Shutdown on download completion"
+msgstr "כבה בסיום ההורדה"
+
+#: youtube_dl_gui/optionsframe.py:342
+msgid "SUDO password"
+msgstr "סיסמת SUDO"
+
+#: youtube_dl_gui/optionsframe.py:420 youtube_dl_gui/optionsframe.py:821
+msgid "In order for the changes to take effect please restart {0}"
+msgstr "על מנת שהשינויים ייכנסו לתוקף אנא הפעל מחדש {0}"
+
+#: youtube_dl_gui/optionsframe.py:421 youtube_dl_gui/optionsframe.py:822
+msgid "Restart"
+msgstr "הפעל מחדש"
+
+#: youtube_dl_gui/optionsframe.py:468
+msgid "high"
+msgstr "גבוה"
+
+#: youtube_dl_gui/optionsframe.py:468
+msgid "low"
+msgstr "נמוך"
+
+#: youtube_dl_gui/optionsframe.py:468
+msgid "mid"
+msgstr "רגיל"
+
+#: youtube_dl_gui/optionsframe.py:473
+msgid "Video formats"
+msgstr "פורמטי וידאו"
```

Comment:
This translates to *Formats formats*, which makes no sense.
Are you sure this is correct?

Reply:
https://translate.google.com/#iw/en/%D7%A4%D7%95%D7%A8%D7%9E%D7%98%D7%99%20%D7%95%D7%99%D7%93%D7%90%D7%95